app/services/analytics.rb

```diff
@@ -76,12 +75,6 @@ def first_event_this_session?
     @session[:first_event]
   end
 
-  def register_doc_auth_step_from_analytics_event(event, attributes)
-    return unless user && user.class != AnonymousUser
-    success = attributes.blank? || attributes[:success] == 'success'
```

Comment:
in hindsight, very strange this code checked that `:success == 'success'` since I'd expect it to be `true|false`